visualization/app/codeCharta/util/actionHelper.ts

```diff
@@ -0,0 +1,3 @@
+export function isActionOfType(actionType: string, actions) {
+	return Object.keys(actions).includes(actionType)
```

Comment:
Why don't you use `Object.values()` kinda like before with `_.values()`?

Reply:
I might look like a rookie, if I say that I didnt know this function existed. That's why I will just resolve this conversation and act like this never happened. Will fix tomorrow.

Reply:
done